cli/cmd/generate_support_archive.go

```diff
@@ -442,96 +442,96 @@ func getIngressGateway(keptnNamespace string) *errorableStringResult {
 
 func writeNamespaces(dir string) {
 	fmt.Println("Retrieving namespaces")
-	writeErrorableStringResult(newErrorableStringResult(exechelper.ExecuteCommand("kubectl", "get namespaces")),
+	writeErrorableStringResult(newErrorableStringResult(keptnutils.ExecuteCommand("kubectl", []string{"get", "namespaces"})),
 		filepath.Join(dir, "namespaces.txt"))
 }
 
 func writeConfigMaps(namespace, dir string) {
 	fmt.Println("Retrieving list of config maps in " + namespace)
-	writeErrorableStringResult(newErrorableStringResult(exechelper.ExecuteCommand("kubectl", "get cm -n "+namespace)),
+	writeErrorableStringResult(newErrorableStringResult(keptnutils.ExecuteCommand("kubectl", []string{"get", "cm", "-n ", namespace})),
```

Comment:
Careful, when you set the parameter to `"-n "`, you end up getting an error message, as the command you generate contains an additional space between `-n` and the namespace, e.g.:
```
Error executing command kubectl get cm -n  keptn: exit status 1
Error from server (NotFound): namespaces " " not found
```